front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ProjectsWidget/ProjectsCard/index.tsx

```diff
@@ -8,8 +8,16 @@ import { Props } from '../typings';
 
 import ProjectRow from './ProjectRow';
 
-const ProjectsCard = ({ startAt, endAt }: Props) => {
-  const { data: response } = useProjects({ start_at: startAt, end_at: endAt });
+const ProjectsCard = ({
+  startAt,
+  endAt,
+  publicationStatus = 'published',
```

Comment:
I'm also not sure about adding this 'fallback' behavior here. I don't think any other widgets do this. It makes it confusing because if you look at the Craft JSON you will see that publication status is undefined, but then suddenly a publication status filter will be applied because this behavior is specified in this widget.

I would rather remove all this default publication status stuff from the FE and move this to the BE (so in the BE say, if no explicit publication status(es) was/were provided, we only show published ones).

Reply:
Easy to do. @EdwinKato let me know if you want my help doing this in a future iteration (though I suspect you are able to do it too)